doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -0,0 +1,564 @@
+==============================
+What's new in Matplotlib 3.6.0
+==============================
+
+For a list of all of the issues and pull requests since the last revision, see
+the :ref:`github-stats`.
+
+.. contents:: Table of Contents
+   :depth: 4
+
+.. toctree::
+   :maxdepth: 4
+
+Figure and Axes creation / management
+=====================================
+
+``subplots``, ``subplot_mosaic`` accept *height_ratios* and *width_ratios* arguments
+------------------------------------------------------------------------------------
+
+The relative width and height of columns and rows in `~.Figure.subplots` and
+`~.Figure.subplot_mosaic` can be controlled by passing *height_ratios* and
+*width_ratios* keyword arguments to the methods. Previously, this required
+passing the ratios in *gridspec_kws* arguments.
+
+New ``layout_engine`` module
+----------------------------
+
+Matplotlib ships with ``tight_layout`` and ``constrained_layout`` layout
+engines.  A new `.layout_engine` module is provided to allow downstream
+libraries to write their own layout engines and `~.figure.Figure` objects can
+now take a `.LayoutEngine` subclass as an argument to the *layout* parameter.
+
```

Comment:
Ack!  Compressed Layout https://github.com/matplotlib/matplotlib/pull/22289 went in with no what's-new!  Would you like me to write one here, or write a separate one?

Reply:
Leave a note here and I will include it as I go through the rest of the PRs.

Reply:
I'll have one tonight. Sorry for the extra bother.